src/hssm/plotting/quantile_probability.py

```diff
@@ -0,0 +1,427 @@
+"""Code for producing quantile probability plots."""
+
+import logging
+from itertools import product
+from typing import Any, Iterable
+
+import arviz as az
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from .utils import (
+    _check_groups_and_groups_order,
+    _get_plotting_df,
+    _get_title,
+    _process_df_for_qp_plot,
+    _subset_df,
+    _use_traces_or_sample,
+)
+
+_logger = logging.getLogger("hssm")
+
+
+def _plot_quantile_probability_1D(
```

Comment:
some of these plotting parameters might be passed through to the plot as **kwargs?
(I know I didn't do this well in HDDM :))

Reply:
reading a bit more I think you already tried to account for that quite a bit, but took the choice e.g. to keep xlabel and ylabel as named arguments here?

Reply:
Sorry just saw this. Yeah I am doing `xlabel` and `ylabel` from experience with R lol. It's hard to decide which ones you explicitly declare and which ones can be included in `kwargs`. I just thought that the users might want to be able to change these labels quickly